web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md

```diff
@@ -6,6 +6,32 @@ hidetitle: True
 #### April 20-22, 2020
 #### Bryn Mawr College, Pennsylvania
 <br/>
+
+<div class="plasmapy-note">
+    <p class="plasmapy-note-title">
+        !! -- Workshop Cancelled
+    </p>
+    <p>
+        Due to the rapid development of COVID-19 (coronavirus) and out of an abundance
+        of caution, the PlasmaPy workshop organizers have decided to cancel PlasmaPy's 
+        first workshop.
+    </p>
+    <p>
+        We originally intended to web-cast portions of the workshop, but, with no 
+        in-person attendees, we feel this format would lead to a poor transmission of 
+        ideas between PlasmaPy and its community.  At this time, it is crucial that 
+        PlasmaPy fosters a strong community interaction to not only to convey its 
+        goals and ambitions, but to receive vital feed back from our community.  
+        <strong>Thus, we have decided to replace this workshop with a webinar 
+        series that will be given over the coming months.</strong>  As more information 
+        on the series becomes available we will update the community.
```

Comment:
```suggestion
        <strong>We have decided to replace this workshop with a series of
        webinars and discussions to be held over the coming months.</strong>
        We will update the community as more information on this series becomes
        available.
```

Reply:
For this one, I'm currently wondering about us having monthly-ish webinars on things like an overview of PlasmaPy and how to contribute, along with two or three community discussions per month on focused topics (diagnostics, transport, simulation, education, HEDP, etc.) where we'd invite people who have a lot of experience on the particular topic so that they could help us with how to plan future development for areas that we do not have expertise in.

Reply:
I agree.  I really like the idea of forming subject specific discussions that we invite experienced researchers to participate in.  I think that would be invaluable to us.

We should probably start migrating the CSSI meetings from being workshop oriented to being webinar oriented.